notebooks/getting_started/ml_fundamentals.ipynb

```diff
@@ -3166,14 +3264,32 @@
    "source": [
     "## Save/Load to BigQuery\n",
     "\n",
-    "Estimators can be saved to BigQuery as BQML models, and loaded again in future"
+    "Estimators can be saved to BigQuery as BQML models, and loaded again in future.\n",
+    "\n",
+    "Saving requires `bigquery.tables.create` permission, and loading requires `bigquery.models.getMetadata` permission.\n",
+    "\n",
+    "If you have those permissions, please go ahead and uncomment the code in the following cells and run."
```

Comment:
👍  Thanks. It is important to continue to document `to_gbq()` here, even if we don't run it.

Maybe clarify that those permissions can be on a project or dataset?

Reply:
Makes sense, done!